src/styles/components/recruit-card.module.scss

```
@@ -147,15 +147,18 @@
 .card__ad__slot {
   width: 100%;
   min-height: 12rem;
+  overflow: auto;
```

Stylelint (17.3.0)

[error] Unexpected duplicate "overflow" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)